feature_server/server.py

```diff
@@ -205,7 +207,16 @@ def on_block_build_attempt(self, x, y, z):
         return self._can_build()
 
     def on_line_build_attempt(self, points):
-        return self._can_build()
+        if not self._can_build():
```

Comment:
Surprisingly, `self._can_build()` [returns `None`](https://github.com/piqueserver/piqueserver/blob/master/feature_server/server.py#L198) if building is allowed. This line should be replaced by `if self._can_build() == False:` to get the desired behavior.